assets/js/app/Data.js

```diff
@@ -183,10 +183,10 @@ LocusZoom.Data.Source.prototype.getData = function(state, fields, outnames, tran
 };
 
 
-LocusZoom.Data.Source.prototype.parseResponse  = function(x, chain, fields, outnames, trans) {
-    var records = this.parseData(x.data || x, fields, outnames, trans);
-    var res = {header: chain.header || {}, body: records};
-    return res;
+LocusZoom.Data.Source.prototype.parseResponse = function(resp, chain, fields, outnames, trans) {
```

Comment:
Ah. OK. I thought that the xmlhttprequest looked at the content type of the response but I see that it does not. Got it. Looks great.